iina/VideoView.swift

```diff
@@ -393,8 +397,45 @@ extension VideoView {
     videoLayer.wantsExtendedDynamicRangeContent = true
     videoLayer.colorspace = CGColorSpace(name: name!)
     mpv.setString(MPVOption.GPURendererOptions.iccProfile, "")
-    mpv.setString(MPVOption.GPURendererOptions.targetTrc, "pq")
     mpv.setString(MPVOption.GPURendererOptions.targetPrim, primaries)
+    // PQ videos will be display as it was, HLG videos will be converted to PQ
+    mpv.setString(MPVOption.GPURendererOptions.targetTrc, "pq")
+    mpv.setFlag(MPVOption.Screenshot.screenshotTagColorspace, true)
```

Comment:
This was used for screenshot in [jxl format](https://github.com/CarterLi/iina/blob/develop/iina/Preference.swift#L538).

Since [(newer) mpv actually doesn't support ppm / pgm ...](https://[mpv.io](https://mpv.io/manual/master/#options-screenshot-format)/manual/master/#options-screenshot-format), I recommand merge my screenshot code too.

Reply:
I had not noticed the old screenshot formats. They were removed from the UI in commit 994d6228202221a8ffccc74c29aa3530ec5366f5 back in 2017. I guess they did not remove the values from the enum to preserve compatibility with old plists. Probably safe to remove them now. I found the IINA+ commit that adds the new formats. I will prepare a separate PR for that change. Thanks for pointing this out.